zxcvbn-test/Properties/AssemblyInfo.cs

```diff
@@ -2,35 +2,10 @@
 using System.Runtime.CompilerServices;
```

Comment:
I think you can safely remove this file

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/commit/f700b1cdd705b88d34aff086db29f48350c392d8

Reply:
I'm sorry, I didn't realize that the `Properties` directory was already there in the original repo. Could you revert your commit?